docs/running-on-kubernetes.md

```diff
@@ -718,6 +718,20 @@ from the other deployment modes. See the [configuration page](configuration.html
     Docker image pull policy used when pulling Docker images with Kubernetes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.limit.cores</code></td>
+  <td>(none)</td>
+  <td>
+    specify the cpu limit for the driver pod
```

Comment:
Looks rest of the docs for config keys all use capitalized full sentences. Can you do the same to be consistent?